ui/pages/remove-snap-account/remove-snap-account.tsx

```diff
@@ -19,32 +31,61 @@ export interface RemoveSnapAccountProps {
 
 const RemoveSnapAccount = ({
   snapId,
-  snapName,
   publicAddress,
 }: RemoveSnapAccountProps) => {
+  const t = useI18nContext();
   return (
     <Box
       className="remove-snap-account-page"
       height={BlockSize.Full}
       width={BlockSize.Full}
-      backgroundColor={BackgroundColor.backgroundDefault}
+      display={Display.Flex}
       borderStyle={BorderStyle.none}
       flexDirection={FlexDirection.Column}
       alignItems={AlignItems.center}
-      paddingBottom={2}
-      paddingTop={2}
+      marginBottom={0}
     >
+      <SnapAuthorshipHeader snapId={snapId} />
       <Box
         display={Display.Flex}
         flexDirection={FlexDirection.Column}
         alignItems={AlignItems.center}
-        height={BlockSize.Full}
+        justifyContent={JustifyContent.center}
+        paddingLeft={4}
+        paddingRight={4}
+        style={{ flexGrow: 1 }}
       >
-        <RemoveSnapAccountContent
```

Comment:
I am curious why you removed the component from these files?

Reply:
I wasn't sure we gained anything from moving the content of the page to a separate component given that the component was quite simple and not reused.